subsys/bluetooth/audio/cap_commander.c

```diff
@@ -77,18 +82,314 @@ int bt_cap_commander_discover(struct bt_conn *conn)
 	return bt_cap_common_discover(conn, cap_commander_discover_complete);
 }
 
+#if defined(CONFIG_BT_BAP_BROADCAST_ASSISTANT)
+static struct bt_bap_broadcast_assistant_cb broadcast_assistant_cb;
+static bool ba_cb_registered;
+
+static void
+copy_broadcast_reception_start_param(struct bt_bap_broadcast_assistant_add_src_param *add_src_param,
+				     struct bt_cap_commander_proc_param *proc_param)
```

Comment:
Consider the following 
```suggestion
copy_broadcast_reception_start_param(struct bt_bap_broadcast_assistant_add_src_param *add_src_param,
				     struct broadcast_reception_start *start_param)
```